README.rst

```diff
@@ -247,6 +247,47 @@ in terms of disk space.
 
 Whether ping checks are created automatically for devices.
 
+``OPENWISP_MONITORING_AUTO_CONFIG_MODIFIED``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
++--------------+-------------+
+| **type**:    | ``bool``    |
++--------------+-------------+
+| **default**: | ``True``    |
++--------------+-------------+
+
+Whether ``config_modified`` checks are created automatically for devices.
+
+``OPENWISP_MONITORING_CONFIG_MODIFIED_MAX_TIME``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
++--------------+-----------+
+| **type**:    |  ``int``  |
++--------------+-----------+
+| **default**: |   ``5``   |
++--------------+-----------+
+
+After ``config`` is ``modified``, if the ``modified`` status does not change  after a
+fixed **duration** then ``device`` health status shall change to ``problem``.
+This fixed **duration** is 5 minutes by defaut and can be changed with the help of this setting.
+The input represents corresponding duration in minutes.
+
+**Note**: If the setting ``AUTO_CONFIG_STATUS`` is disabled then this setting need not be declared.
```

Comment:
What is this AUTO_CONFIG_STATUS setting? Please review that everything makes sense when making changes and submitting PRs